src/server/pfs/server/driver.go

```diff
@@ -2426,71 +2430,109 @@ func (d *driver) getFile(pachClient *client.APIClient, file *pfs.File, offset in
 	if err != nil {
 		return nil, err
 	}
+
+	var (
+		objects    []*pfs.Object
+		blockRefs  []*pfs.BlockRef
+		totalSize  uint64
+		found      bool
+		footer     *pfs.Object
+		prevDir    string
+		pathsFound int
+	)
+
+	// collectReferences adds object storage references to a slice
```

Comment:
Looks like it is called collectMetadata now.

Reply:
Actually I meant to rename it to collectReferences, but forgot to. Lemme know if you have a preference, but I think references makes more sense since object storage references are actually what we are collecting here.